### ISSUE TITLE
svg-aam: what should the implicit role of the <svg> element be?

### ISSUE THREAD

**Issue by dd8** [w3c/aria#2734]
Currently the implicit role of  the `<svg>` element is browser dependent

Chrome: exposed in a11y tree as implicit role=image
Firefox: exposed in a11y tree  as implicit role=graphics-document
Safari: only exposed in a11y tree as implicit role=generic if the element has an accessible name (not exposed otherwise)

Test cases:

```
                <!-- implicit role Chrome role=image, Firefox role=graphics-document, Safari not exposed if no accname  -->
                <svg height="100" width="100" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100">
                <circle cx="50" cy="50" r="40" stroke="black" stroke-width="3" fill="red" />
                </svg>

                <!-- implicit role Chrome role=image, Firefox role=graphics-document, Safari not exposed if no accname  -->
                <svg height="10" width="10" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100">
                <circle cx="50" cy="50" r="40" stroke="black" stroke-width="3" fill="red" />
                </svg>  
                
                <!-- implicit role Chrome role=image, Firefox role=graphics-document, Safari exposed as generic/group if has accname, not exposed if no accname  -->
                <svg height="100" width="100" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100">
                <circle cx="50" cy="50" r="40" stroke="black" stroke-width="3" fill="red" />
                <title>Acc Name</title>
                </svg>

                <!-- implicit role Chrome role=image, Firefox role=graphics-document, Safari exposed as generic/group if has accname, not exposed if no accname  -->
                <svg height="10" width="10" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100">
                <circle cx="50" cy="50" r="40" stroke="black" stroke-width="3" fill="red" />
                <title>Acc Name</title>
                </svg> 
```

**Comment by pkra** [comment]
Possible duplicates: https://github.com/w3c/svg-aam/issues/38 and https://github.com/w3c/aria/issues/2427 and the deep dive last march. (Aside: @cyns @dd8 would you prefer to track svg-aam issues on the main aria repository?)
